src/overlays/actors/ovl_Obj_Driftice/z_obj_driftice.c

```diff
@@ -196,7 +196,7 @@ void func_80A66930(ObjDrifticeStruct2* arg0, ObjDriftice* this, s16* arg2, s16*
     temp_f22 *= arg0->unk_08;
 
     //! FAKE:
-    *arg2 = arg0->unk_00[2] + (s32)((void)0, temp_f22);
+    *arg2 = arg0->unk_04 + (s32)((void)0, temp_f22);
```

Comment:
Hmm, did you check if this is still required with this change? You never know...

Reply:
Flipping the the operands matches